libgalois/include/katana/GraphTopology.h

```diff
@@ -605,6 +607,156 @@ class KATANA_EXPORT BasicTopologyWrapper : public GraphTopologyTypes {
   const Topo* topo_ptr_;
 };
 
+/// filter nodes and edges based on their type values
+/// and creates a new local graph based on the filtered nodes and edges
+/// also maintains mappings from old to new and new to old nodes and edges
```

Comment:
Old and new can be confusing. How about "projected" and "original" instead?

Reply:
sounds good